README.md

```diff
@@ -60,120 +48,159 @@ does not support (directly), repository specific fields in all cases. It's basic
 - RPM
 
 ### Remote repository limitations
-[Remote repository creation](https://www.jfrog.com/confluence/display/JFROG/Repository+Configuration+JSON#RepositoryConfigurationJSON-RemoteRepository) 
-does not support (directly), repository specific fields in all cases. It's basically a union of
+[Remote repository creation](https://www.jfrog.com/confluence/display/JFROG/Repository+Configuration+JSON#RepositoryConfigurationJSON-RemoteRepository) does not support (directly), repository specific fields in all cases. It's basically a union of
 - base remote repo params
 - bower support
 - maven
 - gradle
 - Docker (v1)
 - VCS
 - Pypi
-- Nuget 
+- Nuget
+
 Query params may be forwarded, but this field doesn't exist in the documentation
 
 ### Permission target limitations
 [Permission target V2](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-CreateorReplacePermissionTarget)
-Permission target V1 support has been totally removed. Dynamically testing of permission targets using a new repository
-currently doesn't work because of race conditions when creating a repo. This will have to be resolved with retries at a 
-later date. 
+
+Permission target V1 support has been totally removed. Dynamically testing of permission targets using a new repository currently doesn't work because of race conditions when creating a repo. This will have to be resolved with retries at a later date.
 
 ### Changes to user creation ###
-Previously, passwords were being generated for the user if none was supplied. This was both unnecessary (since TF has a password provider)
-and because the internal implementation could never be entirely in line with the remote server (or, be sure it was).
-Then, with the release of 2.3.1, password was still optional, but if supplied, must watch the default password requirements. These
-could be overridden with `JFROG_PASSWD_VALIDATION_ON=false` if a custom password policy is in place. 
-Now this functionality is removed. Password is a required field. The verification is offloaded to the Artifactory, 
-which makes more sense, so we don't need to catch up with any possible changes on the Artifactory side.
+Previously, passwords were being generated for the user if none was supplied. This was both unnecessary (since TF has a password provider) and because the internal implementation could never be entirely in line with the remote server (or, be sure it was).
 
+Then, with the release of 2.3.1, password was still optional, but if supplied, must watch the default password requirements. These could be overridden with `JFROG_PASSWD_VALIDATION_ON=false` if a custom password policy is in place.
```

Comment:
Did daniel delete this? I believe he was working on something

Reply:
Yes, you're right. We can remove this